aerogear/mobile-developer-console
# feature: Display mdcName label on service instance on bindings page

## Motivation
There was a request to be able to customize the detail name of provisioned service instances. This PR enables that

## How
provisioned services may label their service instance with a label "mdcName" this label will be used on the binding rows as more detail.

## Verification Steps
1. Deploy a sync app and set the sync app name
2. Confirm that the detail on the service binding page is the sync app name and not "Sync App"
